docs/src/modules/components/ApiPage.js

```diff
@@ -255,7 +255,7 @@ export default function ApiPage(props) {
         <PropertiesSection
           properties={componentProps}
           propertiesDescriptions={propDescriptions}
-          targetName={pageContent.name}
+          componentName={pageContent.name}
```

Comment:
I went with a solution that minimize the number of types that we manipulate. It seems simpler to kebab case convert at the very end.